doc/advanced_configuration.rst

```diff
@@ -520,20 +522,118 @@ The highest precedence is always given to the `-D` flag of the
 ``sphinx-build`` command.
 
 
-.. _find_mayavi:
+.. _image_scrapers:
 
-Finding Mayavi figures
-======================
-By default, Sphinx-gallery will only look for :mod:`matplotlib.pyplot` figures
-when building. However, extracting figures generated by :mod:`mayavi.mlab` is
-also supported. To enable this feature, you can do::
+Image scrapers
```

Comment:
I would like to add that this is an Experimental feature. Here in the Title and in the description. This is something we are trying out in order to capture different output objects

Reply:
IIRC @lesteve wanted such a warning if we were not satisfied with the API. Do you think that it might need to change?